### PR TITLE
Add Date of Birth field 

### PR DESCRIPTION
**Pull request by Pujitha97**
Along with relevant document and tests

### CODE REVIEW

File: docs/AboutUs.adoc [hunk 1]
```diff
@@ -17,6 +17,13 @@ image:githubicon.png[Github Icon, 40, 40, link="https://github.com/OscarWang114"
 Role: Developer +
 Responsibilities:
 
+=== Pujitha Desiraju
+image::PujithaDesiraju.jpg[width="150", align="left"]
```
Comment (OscarWang114) [inline docs/AboutUs.adoc:21]: Please change the extension of the added file PujithaDesiraju.JPG to jpg (I cannot review on that so I did it here). (I'm being super picky :))

**Comment by OscarWang114** [comment]
Have you tried running the program on IntelliJ? If it's successful that means your data addressbook.xml is updated with dob info :) (You probably changed it yourself?) 

I want to test your code and require the updated addressbook.xml under the data directory. Currently, it is ignored by Git. Can you disable the ignore temporarily and commit again?

**Comment by OscarWang114** [comment]
Just found out the PR prior to yours caused some test cases to fail on my MacBook. 
I'm assuming it's due to keyboard shortcuts. I'll create an issue and come back reviewing after solving the problem.

File: docs/AboutUs.adoc (hunk 1 above)
Comment (Pujitha97) [reply docs/AboutUs.adoc:21]: Hey , i have run my program on intelliJ and it passed all 246 tests. I can commit the file as well. Thanks for the review :)

Comment (OscarWang114) [reply docs/AboutUs.adoc:21]: Nice! Including MainApp right? 
Yes please do commit the updated addressbook.xml so everyone else can run MainApp successfully :)

**Pull request closed without merging**